spec/sms_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'sms'
+
+describe SMS do
+	subject(:sms){ described_class.new(config, client:client) }
+
+	let(:client){ double(:client, messages:messages) }
+	let(:messages){ spy(:messages) }
+
+	let(:config) do 
+		{
+			account_sid: "C3462f5dec",
+			auth_token: "01d58",
+			from: "+440750",
+			to: "+4407501",
+			body: "Thank you! Your order was placed and will be delivered before %s"
+		}
+	end
```

Comment:
This config looks pretty cumbersome to use. Ideally, the only input I would need to send a message would be the telephone number and the body.